protected/humhub/modules/content/widgets/stream/WallStreamEntryWidget.php

```diff
@@ -139,6 +145,16 @@ abstract class WallStreamEntryWidget extends StreamEntryWidget
      */
     protected $renderOptionClass = WallStreamEntryOptions::class;
 
+    /**
+     * @var string
+     */
+    public $menuAction;
```

Comment:
Add Docs. When action for creation, please add word `createForm`

Reply:
Commit https://github.com/humhub/humhub/pull/5827/commits/33cca19cc872d6bdde229d426b2d47a60b82fb74